### ISSUE TITLE
Ingamemeldungen (feature request) : Statt Fahrzeugname das Nummernschild ausgeben

### ISSUE THREAD

**Issue by Bliro** [Stephan-S/FS22_AutoDrive#275]
Die Meldung "Fahrzeug xy" ist bei "z" angekommen ist wirklich ein super hilfreiches Feature.

Im Multiplayer ist es jedoch nicht so ganz einfach, auf Anhieb heraus zu finden wem das Fahrzeug gehört.
Könnte man bei der Meldung evtl zB das Nummernschild mit auslesen lassen um so einen schnelleren Überblick zu haben ob das Fahrzeug einen selber betrifft ?

Alternativ die Meldungen anderer Höfe (ohne Lohnarbeiterzugriff) nicht mehr anzeigen.

**Comment by Iwan1803** [comment]
Schon mal dran gedacht, das man mit AD Fahrzeuge benennen kann?
Dazu muss man nur in den LS22-Tastenbelegungen eine Kombi belegen und fertig 😉

**Comment by Bliro** [comment]
nope, die Funktion war mir noch nicht ins Auge gehüpft. werde ich mal nach schauen

**Comment by c0nnex** [comment]
Die Meldungen von Fahrern anderrer Höfe nicht merh anzuzeigen wäre trotzdem ganz sinnvoll.
Ich will die meldungen nicht abschalten, sonst verlier ich den Überblick, werd aber von den andern zugespammt :(
